content/src/display-content/contextual-information/tenant-improvements.md

```diff
@@ -7,4 +7,4 @@ header: Tenant Improvements
 
 Few spaces are "turnkey ready" for your business – most businesses will require some changes and customization to help them realize their vision. Depending on the space and the market, you may be able to negotiate Tenant Improvements (or "TIs") with your landlord. Tenant Improvements can take the form of an allowance to help make improvements to the space, free rent, or the landlord may agree to make the changes themselves.
 
-Related information: `ADA Compliant Property|ada-compliant`
```

Comment:
Feels super weird to have such a tiny story.

Worried I may have done this wrong somehow? Would appreciate reviewers reading the AC to make sure

Also validating my approach (which was simply control + f to look for all instances of ada-complaint and this was the only one, nothin else here I should be doing to look for instances right?)

Reply:
this looks right, I did a search on my end and only found this one instance